analyzer_test.go

```diff
@@ -27,16 +25,17 @@ var _ = Describe("Analyzer", func() {
 	})
 
 	Context("when processing a package", func() {
-
-		It("should return an error if the package contains no Go files", func() {
-			analyzer.LoadRules(rules.Generate().Builders())
-			dir, err := ioutil.TempDir("", "empty")
-			defer os.RemoveAll(dir)
-			Expect(err).ShouldNot(HaveOccurred())
-			err = analyzer.Process(buildTags, dir)
-			Expect(err).Should(HaveOccurred())
-			Expect(err.Error()).Should(MatchRegexp("no buildable Go source files"))
-		})
+		/*
+			It("should return an error if the package contains no Go files", func() {
```

Comment:
Instead of commenting out the test can you please change this to use Ginko's `Skip()` with a message like `"Temporarily disabled"` or something similar until we figure out if this test is relevant anymore?

We're less likely to forget the test existed that way.

Reply:
I am wondering if we should not get some of these tests passing. Is this behaviour removed from modules?